README.md

```diff
@@ -33,8 +33,7 @@ Our projects are community-built and welcome collaboration. 👍 Be sure to see
 
 ✔️ <em><strong>Join</strong></em> any or all of the weekly meetings on the <a href="https://calendar.google.com/calendar/b/1?cid=bGF5ZXI1LmlvX2VoMmFhOWRwZjFnNDBlbHZvYzc2MmpucGhzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20">community calendar</a>.<br />
 ✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">meeting recordings</a>.<br />
-✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">Community Drive</a> by completing a community <a href="https://layer5.io/newcomer">Member Form</a>.<br />
-</p>
+✔️ <em><strong>To access the community drive</strong></em>, please fill out the <a href="<form-link>">Newcomer's form</a>.<br />
```

Comment:
@amino19 There's no form link only 😶

Reply:
Replace `"<form-link>"` in Line 6 with `"https://layer5.io/newcomer"`

Reply:
It's with an 's'
`https://layer5.io/newcomers`

Reply:
@Nikhil-Ladha the  one with s gives 404 Error

Reply:
Ohh, okay 😂